entity-framework/toc.yml

```diff
@@ -175,6 +175,10 @@
       items:
       - name: Overview
         href: core/querying/index.md
+      - name: Client vs. server evaluation
```

Comment:
I think this undoes a change we recently made... FWIW I think loading related data is more immediately important than both client/server evaluation (definitely since 3.0) and tracking/no-tracking.

Reply:
I will think about it. I do not agree with that priority order.

Reply:
Either way this should be done in a separate PR to keep discussion on-topic